CONTRIBUTING.md

```diff
@@ -176,7 +176,14 @@ See the [`anndata2ri` docs](https://icb-anndata2ri.readthedocs-hosted.com/en/lat
 
 ### Adding package dependencies
 
-If you are unable to write your method using our base dependencies, you may add to our existing Docker images, or create your own. The image you wish to use (if you are not using the base image) should be specified in the `image` keyword argument of the method/metric decorator. See the [Docker images README](docker/README.md) for details.
+If you are unable to write your method using our base dependencies, you may add to our existing Docker images, or create your own. The image you wish to use (if you are not using the base image) should be specified in the `image` keyword argument of the method/metric decorator. See the [Docker images README](docker/README.md) for details. 
+
+For the target method or metric, image-specific methods can then be imported in the associated python file (i.e. `f2.py`). Importantly, the import statement must be located in the function that calls the method or metric. If this is the `f2` function, the first few lines of the function may be structured as follows:
```

Comment:
image-specific methods -> image-specific packages and then I think we're good to go!